components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -971,9 +968,7 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th,qv,dt,qitot,qirim,nitot,birim,   &
 
           !.................................................................
           ! droplet activation
-          call droplet_activation(t(i,k),pres(i,k),qv(i,k),qc(i,k),inv_rho(i,k),&
-             sup(i,k),xxlv(i,k),npccn(i,k),log_predictNc,odt,&
-             qcnuc,ncnuc)
+          ! The macrophysics handles the activation to cloud drops number and mass (see 'ncnuc')
```

Comment:
We should probably just completely delete this mention of droplet activation here.